mappings/net/minecraft/client/gui/screen/world/CreateWorldScreen.mapping

```diff
@@ -1,57 +1,112 @@
 CLASS net/minecraft/unmapped/C_ibkofvzc net/minecraft/client/gui/screen/world/CreateWorldScreen
 	FIELD f_bkgndroc ALLOW_COMMANDS_INFO_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_bqfuezdy reopened Z
+	FIELD f_bwrlwywb GROUP_BOTTOM I
 	FIELD f_fyvyzcba PREPARING_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_hqkhcpia header Lnet/minecraft/unmapped/C_mbxnkehf;
 	FIELD f_htfhyyfx packManager Lnet/minecraft/unmapped/C_dqfyacfy;
 	FIELD f_hxpwlzvr ENTER_NAME_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_jlpvoazq FOOTER_SEPARATOR_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_lryffkzq LIGHT_DIRT_BACKGROUND_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_lzdtplvi HEADER_SEPARATOR_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_ntbjtmej TAB_COLUMN_WIDTH I
+	FIELD f_ohynjwms FOOTER_HEIGHT I
 	FIELD f_puiuugkf EXPERIMENTS_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_puwignwj HORIZONTAL_BUTTON_SPACING I
+	FIELD f_rsmdhran worldCreator Lnet/minecraft/unmapped/C_hypraqfe;
 	FIELD f_toffzjsn parent Lnet/minecraft/unmapped/C_wrmtlwqx;
 	FIELD f_trqmzvlv TEMP_DIR_PREFIX Ljava/lang/String;
+	FIELD f_vgcizakt TEXT_INDENT I
 	FIELD f_vqvehnjz LOGGER Lorg/slf4j/Logger;
+	FIELD f_vsdozuic VERTICAL_BUTTON_SPACING I
 	FIELD f_woybhigv dataPackTempDir Ljava/nio/file/Path;
 	FIELD f_wuyzkrbd GAME_MODE_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_xabhnhby grid Lnet/minecraft/unmapped/C_qykucwmu;
+	FIELD f_ztgjtcrr tabManager Lnet/minecraft/unmapped/C_shidhwhk;
 	METHOD <init> (Lnet/minecraft/unmapped/C_ayfeobid;Lnet/minecraft/unmapped/C_wrmtlwqx;Lnet/minecraft/unmapped/C_njsjipmy;Ljava/util/Optional;Ljava/util/OptionalLong;)V
 		ARG 2 parent
+		ARG 3 context
+		ARG 4 typeKey
+		ARG 5 seed
 	METHOD m_bmhqnfoo (Lnet/minecraft/unmapped/C_dqfyacfy;)V
 		ARG 1 manager
+	METHOD m_btwtdkmu validateDatapacks (Lnet/minecraft/unmapped/C_dqfyacfy;Lnet/minecraft/unmapped/C_yknpgzdr;Ljava/util/function/Consumer;)V
```

Comment:
hm, I wonder if `datapack` is on the word list, anyway, this should instead be:
```suggestion
	METHOD m_btwtdkmu validateDataPacks (Lnet/minecraft/unmapped/C_dqfyacfy;Lnet/minecraft/unmapped/C_yknpgzdr;Ljava/util/function/Consumer;)V
```

Reply:
![image](https://github.com/QuiltMC/quilt-mappings/assets/85403855/3afd57ef-d5c9-4f06-be1b-ad50fec9ff03)

it is (line 4), should it be removed?

Reply:
It can be safely removed, with this use on `MinecraftServer` being the only spot that uses it
![Screenshot of the only instance of "datapack" being used in mappings](https://github.com/QuiltMC/quilt-mappings/assets/85590273/1472dd9a-7bdd-418b-8d24-6fe7e0d32df7)

Reply:
oh no, i didn't check against `Datapack`; there are two classes to be refactored; an another PR would be better

Reply:
> oh no, i didn't check against `Datapack`; there are two classes to be refactored; an another PR would be better

...except not! :woman_facepalming:
`DatapackFailureScreen` should be renamed to `DataPackFailureScreen`, `DatapackCommand` needs to be like that because the command is `/datapack`, therefore, yeah, it's more complicated than expected

Reply:
> > oh no, i didn't check against `Datapack`; there are two classes to be refactored; an another PR would be better
> 
> ...except not! woman_facepalming `DatapackFailureScreen` should be renamed to `DataPackFailureScreen`, `DatapackCommand` needs to be like that because the command is `/datapack`, therefore, yeah, it's more complicated than expected

...and nvm, `datapack` can be removed because considering `/playsound` becomes `PlaySoundCommand`, the command can reasonably be renamed to `DataPackCommand`